Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -688,3 +688,85 @@ label mas_wrs_reddit:
     if not wrs_success:
         $ mas_unlockFailedWRS('mas_wrs_reddit')
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_mal",
+            category=['myanimelist'],
+            rules={"notif-group": "Window Reactions", "skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_mal:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "So you like anime and manga, [player]?",
```

Comment:
There's a player model variable for this, so we should actually be able to know whether or not the player likes this.